### PR TITLE
revert INC fixed version installation in requirements-hpu.txt for 1.19, add tmp one for 1.18

### PR DESCRIPTION
**Pull request by xuechendi**
Fix 1.19 INC incompatible issues

https://jira.habana-labs.com/browse/SW-209688

### CODE REVIEW

**Review by michalkuligowski — CHANGES_REQUESTED** [review]
Lets not confuse users with adding new requirements file

File: requirements-hpu.txt [hunk 1]
```diff
@@ -9,4 +9,3 @@ tabulate
 setuptools>=61
 setuptools-scm>=8
 vllm-hpu-extension @ git+https://github.com/HabanaAI/vllm-hpu-extension.git@ac9740d
-neural-compressor @ git+https://github.com/intel/neural-compressor.git@b196432
```
Comment (michalkuligowski) [inline requirements-hpu.txt:12]: Approved

File: requirements-hpu-1.18.txt [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Common dependencies
```
Comment (michalkuligowski) [inline requirements-hpu-1.18.txt:1]: Lets not add this, there is description in issues and release notes on how to handle this issue.

**Review by michalkuligowski — CHANGES_REQUESTED** [review]

**Review by michalkuligowski — APPROVED** [review]

**Comment by michalkuligowski** [comment]
@xuechendi please resolve conflict

**Comment by xuechendi** [comment]
@michalkuligowski , conflict resolved.

**Pull request merged**